src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -172,6 +172,8 @@ java.launcher.X.usage=\n\
 \                      (Linux Only) show host system or container\n\
 \                      configuration and continue\n\
 \    -Xss<size>        set java thread stack size\n\
+\                      The actual size may be round up to multiple of system\n\
```

Comment:
See updated help text in the CSR request.

Reply:
Updated, thanks